cassandra2/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -0,0 +1,447 @@
+/**
+ * Copyright (c) 2013 Yahoo! Inc. All rights reserved.
```

Comment:
Mostly identical, other than the method call to shut down a `Cluster`/`Session`. Updated copyright.